tailwind.config.js

```diff
@@ -14,25 +14,25 @@ module.exports = {
         base: '14px',
       },
       maxWidth: {
-        '7xl': '1260px',
+        '7xl': '1128px',
       },
       colors: {
-        primary: '#154EC1',
-        secondary: '#767C88',
-        success: '#3D9209',
-        info: '#158DE3',
-        danger: '#E42222',
-        warning: '#FFD43A',
-        backgroundPrimary: '#f6f6f6',
-        backgroundSecondary: '#FFFFFF',
-        backgroundElement: '#ECF0F1',
-        backgroundCardPrimary: '#F7F9F9',
-        backgroundCardSecondary: '#ECFDE6',
-        backgroundBorder: '#DEE5F2',
-        textPrimary: '#262824',
-        textInverted: '#FFFFFF',
-        shadow: 'rgba(0, 0, 0, 0.12)',
-        focus: '#49A8FF',
+        primary: 'var(--va-primary)',
```

Comment:
That's interesting! Are you sure this works fine?

If yes - we probably have to change tailwind sync script as it's not doing great right now. (could absolutely be a different issue)